src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -67,6 +71,20 @@
     public static final String TAG_DESC_FRIEND = " " + PREFIX_TAG + VALID_TAG_FRIEND;
     public static final String TAG_DESC_HUSBAND = " " + PREFIX_TAG + VALID_TAG_HUSBAND;
 
+    public static final String POSITION_DESC_JR_SWE = " " + PREFIX_POSITION + VALID_POSITION_NAME_JR_SWE;
+    public static final String POSITION_DESC_SR_FE_SWE = " " + PREFIX_POSITION + VALID_POSITION_NAME_SR_FE_SWE;
+    public static final String DESCRIPTION_DESC_JR_SWE = " " + PREFIX_DESCRIPTION + VALID_DESCRIPTION_NAME_JR_SWE;
+    public static final String DESCRIPTION_DESC_SR_FE_SWE = " " + PREFIX_DESCRIPTION + VALID_DESCRIPTION_NAME_SR_FE_SWE;
+    public static final String OPENING_DESC_JR_SWE = " " + PREFIX_NUM_OPENINGS + VALID_POSITION_OPENINGS_JR_SWE;
+    public static final String OPENING_DESC_SR_FE_SWE = " " + PREFIX_NUM_OPENINGS + VALID_POSITION_OPENINGS_SR_FE_SWE;
+    public static final String REQUIREMENT_DESC_EXPERIENCE = " " + PREFIX_REQUIREMENT + VALID_REQUIREMENT_EXPERIENCE;
+    public static final String REQUIREMENT_DESC_SKILL = " " + PREFIX_REQUIREMENT + VALID_REQUIREMENT_SKILL;
+
+    public static final String INVALID_POSITION_DESC = " " + PREFIX_POSITION + "^#$#"; // only special characters
+    public static final String INVALID_DESCRIPTION_DESC = " " + PREFIX_DESCRIPTION + "  "; // on whitespaces
+    public static final String INVALID_OPENING_DESC = " " + PREFIX_NUM_OPENINGS + "-1"; // negative integer
+    public static final String INVALID_REQUIREMENT_DESC = " " + PREFIX_REQUIREMENT + "*!@#"; // only special characters
```

Comment:
Perhaps you would like to change the comments here? Maybe something like no special characters allowed in requirement